### ISSUE TITLE
Add Orca if it ever releases

### ISSUE THREAD

**Issue by timjzee** [opening-up-chatgpt/opening-up-chatgpt.github.io#47]
Whitepaper: https://arxiv.org/pdf/2306.02707.pdf

Will be released here: https://aka.ms/orca-lm

Summary: https://www.youtube.com/watch?v=Dt_UNg7Mchg

**Comment by mdingemanse** [comment]
Interesting find. And more evidence of the growing importance of synthetic instruction tuning data.

**Comment by mdingemanse** [comment]
Looks like there may be some version of it here: https://huggingface.co/yhyhy3/med-orca-instruct-33b-GPTQ

**Comment by timjzee** [comment]
I think most of the "Orca" models on Huggingface are projects which used a similar approach to the one described in the Microsoft paper. AFAIK they are not actual Orca releases.

**Comment by mdingemanse** [comment]
Nah. There is a [new preprint](https://arxiv.org/pdf/2311.11045.pdf) that says
>We open-source Orca 2 to encourage further research on the development, evaluation, and alignment of smaller LMs.
But nothing is open-sourced; this is a Llama2 finetune where only the instruction-tuned (or what they call explanation-tuned) model weights are made available, but none of instruction/explanation datasets and none of the source code is made available. 

Thanks Meta for thoroughly diluting the term open source and thanks Microsoft for further contributing to it.

**Comment by mdingemanse** [comment]
Nope
